client/src/App.js

```diff
@@ -143,24 +131,28 @@ class App extends Component {
         <div className={classes.root}>
           <AppBar position="static">
             <Toolbar className="theme">
-              <Typography variant="h6" color="inherit" className={classes.grow}>
-                Project GreenBelt
+              <Typography variant="h5" color="inherit"  className={classes.grow}>
+                Greentoad
```

Comment:
This should have Green**T**oad capitalized.

Reply:
Done